scheduler/test/cook/test/mesos/monitor.clj

```diff
@@ -50,7 +52,10 @@
     (is (= 0 (counter ["hungry" "users"])))
     (is (= 0 (counter ["satisfied" "users"])))
 
-    (api/create-jobs! conn {::api/jobs [job1 job2]})
+    (with-redefs
+      [job-submission-rate-limiter ratelimit/NOOP-RateLimiter]
+      (api/create-jobs! conn {::api/jobs [job1 job2]}))
+
```

Comment:
Why not use the wrapper helper function?

Reply:
The one I did in cook.test.mesos.api? I wanted the tests to be self-contained. To re-use that two lines of code would require refactoring the two lines of code into a common shared file (or having one unit test file ns :require the other. That makes both tests files less self-contained, and harder to read as you can't follow the code, you have to trace it.

Reply:
`testutil`?